packages/expo-cli/src/commands/build-native/index.ts

```diff
@@ -70,12 +73,136 @@ export async function buildAction(projectDir: string, options: Options): Promise
   }
 }
 
+async function logBuildStatuses(
```

Comment:
Please move the new functions to a separate file. We're trying to keep only the command definition in the command files.